lambdas/securty-headers.js

```diff
@@ -0,0 +1,80 @@
+exports.handler = (event, _, callback) => {
+  const response = event.Records[0].cf.response;
+
+  Object.assign(response.headers, {
+    // https://infosec.mozilla.org/guidelines/web_security#http-strict-transport-security
+    'strict-transport-security': [
+      {
+        key: 'Strict-Transport-Security',
+        value: 'max-age=10886400000; includeSubdomains; preload',
+      },
+    ],
+
+    // https://infosec.mozilla.org/guidelines/web_security#content-security-policy
+    'content-security-policy': [
+      {
+        key: 'Content-Security-Policy',
+        value: [
+          "default-src 'none'",
+          "connect-src 'self'",
```

Comment:
This allows the app to make POST requests to the API domains

```suggestion
          "connect-src 'self' https://*.plaid.com https://*.plaid.io",
```

Reply:
done

Reply:
done